### PR TITLE
[zrx-voting-power] ZRX voting power strategy

### PR DESCRIPTION
**Pull request by lyaunzbe**
Hello! I work with 0x labs on the platform side. I am making this PR to hopefully add the zrx-voting-power strategy (calculates voting power based on staked + wallet balances). Thank you!

Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

**Review by OhArthits — APPROVED** [review]

File: src/strategies/zrx-voting-power/index.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      {
+        internalType: 'uint256',
+        name: 'votingPower',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/zrx-voting-power/index.ts:42]:
```suggestion
const abi = [
  'function getVotingPower(address account, bytes32[] operatedPoolIds) view returns (uint256 votingPower)'
]
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/zrx-voting-power/index.ts (hunk 1 above)
Comment (lyaunzbe) [reply src/strategies/zrx-voting-power/index.ts]: thank you, i've made this change

**Comment by ChaituVR** [comment]
@lyaunzbe there is conflict, can u resolve it plz :) everything else looks good to me

**Comment by lyaunzbe** [comment]
@ChaituVR should be all set now, thank you!

File: src/strategies/zrx-voting-power/index.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag =
+    typeof snapshot === 'number'
+      ? snapshot
+      : await provider.getBlockNumber(snapshot);
```
Comment (bonustrack) [inline src/strategies/zrx-voting-power/index.ts:35]: You don't need to query for current block number here, 'latest' will work.
```suggestion
      : 'latest';
```

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**